compiler-plugin/src/main/java/io/ballerina/stdlib/persist/compiler/model/RelationField.java

```diff
@@ -116,4 +120,12 @@ public RelationType getRelationType() {
     public void setRelationType(RelationType relationType) {
         this.relationType = relationType;
     }
+
+    public NodeList<AnnotationNode> getAnnotations() {
+        return annotations;
+    }
+
+    public void setAnnotations(NodeList<AnnotationNode> annotations) {
```

Comment:
We are passing the `annotations` as a constructor parameter itself. Is there a case where this annotations need to be updated after initialization of the `RelationField` instance. If not `setAnnotations` method is not needed.